### DISCUSSION TITLE
Lack of Response from GitHub for 2 Months

### DISCUSSION THREAD

**Discussion by Kusieq** [community/community#126751] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My account was flagged two months ago and I still haven't received a response. What can I do to get an answer? Wait another few months?
Any ideas?

**Comment by ebndev** [comment]
Hi @Kusieq, thank you for your patience.

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
